docs/source/nlp/tutorials/quickstarts/plot_multi_label_classification.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
+"""
+NLP Multi Label Classification Quickstart
+*****************************************
+
+In this quickstart guide, we will go over using the deepchecks NLP package to analyze and evaluate a text
+multi label classification task. We will cover the following:
+
+1. `Creating a TextData object and auto calculating properties <#setting-up>`__
+2. `Running the built-in suites <#running-the-deepchecks-default-suites>`__
+3. `Running individual checks <#running-individual-checks>`__
+
+To run deepchecks for NLP, you need the following for both your train and test data:
+
+1. Your text data - a list of strings, each string is a single sample (can be a sentence, paragraph, document, etc.).
+2. Your labels and prediction in the :ref:`correct format <nlp__supported_text_classification>` (Optional).
+3. :ref:`Metadata <nlp__metadata_guide>`, :ref:`Properties <nlp__properties_guide>`
+   or :ref:`Embeddings <nlp__embeddings_guide>` for the provided text data (Optional).
+
+If you don't have deepchecks installed yet:
+
+.. code:: python
+
+    import sys
+    !{sys.executable} -m pip install deepchecks[nlp] -U --quiet #--user
+
+Some properties calculated by ``deepchecks.nlp`` require additional packages to be installed. You can
+install them by running:
+
+.. code:: python
+
+    import sys
+    !{sys.executable} -m pip install [nlp-properties] -U --quiet #--user
+
+Setting Up
+==========
+
+Load Data
+---------
+For the purpose of this guide, we'll use a small subset of the
+`just dance <https://www.kaggle.com/datasets/renatojmsantos/just-dance-on-youtube>`__ comment analysis dataset.
+Dataset containing comments, metadata and labels for multilabel category classification of the comments.
+
+"""
+
+from deepchecks.nlp import TextData
+from deepchecks.nlp.datasets.classification import just_dance_comment_analysis
+
+data = just_dance_comment_analysis.load_data(data_format='DataFrame', as_train_test=False)
+metadata_cols = ['likes', 'dateComment']
+data.head(2)
+
+# %%
+# Create TextData Objects
+# ------------------------
+#
+# Deepchecks' :ref:`TextData <nlp__textdata_object>` object contains the text samples, labels, and possibly
```

Comment:
in the boilerplate pdf I can't see if this link works, does it?
(btw to where does it point? cause it's not to the api reference. anyway - should be consistent across quickstarts, and point to the most relevant file :)

Reply:
This is the text data guide I wrote. I think it's actually better than linking to the class, updating it also in my quickstart.